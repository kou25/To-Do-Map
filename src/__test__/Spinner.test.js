import React from 'react';
import ReactDom from 'react-dom'
import Spinner from '../containers/view/Spinner';
import renderer from 'react-test-renderer'
import {cleanup} from '@testing-library/react'

afterEach(cleanup)
it("render without crashing",()=>{
  const div= document.createElement("div");
  ReactDom.render(<Spinner/>,div);
})

it("image render without crashing",()=>{
    const img = document.createElement("img");
    ReactDom.render(<Spinner/>,img);
  })
  

it('matches snapshot', ()=>{
  const tree=renderer.create(
  <Spinner/>).toJSON()
  expect(tree).toMatchSnapshot()
})
