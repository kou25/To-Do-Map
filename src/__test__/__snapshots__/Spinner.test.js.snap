// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <img
    alt="Loadng..."
    src="spinner.gif"
    style={
      Object {
        "display": "block",
        "justifyContent": "center",
        "margin": "auto",
        "width": "200px",
      }
    }
  />
</div>
`;
